test/distrib/cpp/run_distrib_test_cmake_fetchcontent.sh

```diff
@@ -21,12 +21,6 @@ grpc_dir=$(pwd)
 # Install openssl (to use instead of boringssl)
 apt-get update && apt-get install -y libssl-dev
 
-# Install CMake 3.16
```

Comment:
Is the cmake version we need already installed?

Reply:
Yep. All docker images for tests have CMake 3.16 or later.